### ISSUE TITLE
Wafris 2.0 middleware error: FrozenError - can't modify frozen String: ""

### ISSUE THREAD

**Issue by baueric** [Wafris/wafris-rb#80]
After upgrading to Wafris 2.0 and launching in development with my wafris api key I see this log printed on every request:

```log
[Wafris] [Wafris] Detailed Error: FrozenError - can't modify frozen String: ""
[Wafris] [Wafris] Backtrace: .rbenv/versions/3.2.4/lib/ruby/gems/3.2.0/gems/wafris-2.0.0/lib/wafris/middleware.rb:54:in `force_encoding'
.rbenv/versions/3.2.4/lib/ruby/gems/3.2.0/gems/wafris-2.0.0/lib/wafris/middleware.rb:54:in `block in call'
.rbenv/versions/3.2.4/lib/ruby/gems/3.2.0/gems/wafris-2.0.0/lib/wafris/middleware.rb:54:in `each'
.rbenv/versions/3.2.4/lib/ruby/gems/3.2.0/gems/wafris-2.0.0/lib/wafris/middleware.rb:54:in `each_with_object'
.rbenv/versions/3.2.4/lib/ruby/gems/3.2.0/gems/wafris-2.0.0/lib/wafris/middleware.rb:54:in `call'
```
![Screenshot 2024-08-26 at 11 14 44](https://github.com/user-attachments/assets/95265692-fbf8-4671-b646-b0bb30a8e6a4)

**Comment by rmcastil** [comment]
@baueric let me take a look. We were at Laracon all last week so catching up on all the emails.

**Comment by rmcastil** [comment]
Closed as this is solved with the v2.01 release via #83. Thanks again for the PR @baueric!
